conans/client/installer.py

```diff
@@ -123,27 +123,17 @@ def _build_package(self):
                       self._conan_file, self._out)
         self._out.info('Copying sources to build folder')
 
-        def check_max_path_len(src, files):
-            if platform.system() != "Windows":
-                return []
-            filtered_files = []
-            for the_file in files:
-                source_path = os.path.join(src, the_file)
-                # Without storage path, just relative
-                rel_path = os.path.relpath(source_path, src_folder)
-                dest_path = os.path.normpath(os.path.join(self.build_folder, rel_path))
-                # it is NOT that "/" is counted as "\\" so it counts double
-                # seems a bug in python, overflows paths near the limit of 260,
-                if len(dest_path) >= 249:
-                    filtered_files.append(the_file)
-                    self._out.warn("Filename too long, file excluded: %s" % dest_path)
-            return filtered_files
-
         if getattr(self._conan_file, 'no_copy_source', False):
             mkdir(self.build_folder)
             self._conan_file.source_folder = src_folder
         else:
-            shutil.copytree(src_folder, self.build_folder, symlinks=True, ignore=check_max_path_len)
+            if platform.system() == "Windows" and os.getenv("CONAN_USER_HOME_SHORT") != "None":
```

Comment:
!= "None" from the os environment doesn't feels good. If it's happening probably we are doing something wrong.

Reply:
It is the convention I have followed, if the user sets CONAN_USER_HOME_SHORT = None (None as text, as all env-vars). Why doesn't feel good?